### DISCUSSION TITLE
poly delete

### DISCUSSION THREAD

**Discussion by noelmcloughlin** [DavidVujic/python-polylith#168] [Ideas]
**Is your feature request related to a problem? Please describe.**

Poly command supports diff/info/libs/sync/check/create but not delete.

**Describe the solution you'd like**

Could delete be added?

**Describe alternatives you've considered**

For now I use `rm -fr` to remove directory.  If that is correct solution maybe docs could be updated

**Comment by DavidVujic** [comment]
Thank you @noelmcloughlin for the suggestion and ideas! ⭐ 

I will give this some thought! I think a `delete` command would also mean cleaning up the removed item (a brick?) from the projects too, maybe?

Currently, the target is to align with the features of the fellow Clojure tooling. But I like your idea! 

(Moving this to the discussions section)
